src/components/MainWrapper/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex"
+  },
+  content: {
+    margin: "auto",
+    padding: theme.spacing(3),
+    maxWidth: "40%",
+    [theme.breakpoints.down("md")]: {
+      maxWidth: "50%"
+    },
+    [theme.breakpoints.down("sm")]: {
+      maxWidth: "80%"
+    }
+  },
+  extraWide: {
+    margin: "auto",
+    padding: theme.spacing(3),
+    maxWidth: "50%",
+    [theme.breakpoints.down("md")]: {
+      maxWidth: "67%"
+    },
+    [theme.breakpoints.down("sm")]: {
+      maxWidth: "80%"
+    }
+  }
+}));
+
+export const MainWrapper = ({ children, styleType = "default" }) => {
```

Comment:
You could have the default value as "content" then you wouldn't need that ternary you could just do <`main className={classes[styleType]}>{children}</main> `. What do you think?

Reply:
I think that's really clever :-) I've updated it as you suggested.